res/overwatch.example.json

```diff
@@ -1,4 +1,7 @@
 {
+    "timestamp": 0,
+    "platform": "pc",
```

Comment:
Can you move platform and region to have a key under each server in servers? As this ramps up, each team could be located in a different region/ on a different platform

Reply:
Ok this has been updated to include platform, region in each server